src/object/tests/SConscript

```diff
@@ -11,6 +11,8 @@ def scons():
 
     unit_env.Append(RPATH_FULL=['$PREFIX/lib64/daos_srv'])
     unit_env.Append(CPPDEFINES=['-DDAOS_PMEM_BUILD'])
+    unit_env.AppendUnique(RPATH_FULL=['$PREFIX/lib64/daos_srv'])
```

Comment:
Is this not the same as line 12?   I mean AppendUnique is generally preferred but line 12 would have appended it already so this line is a nop.